Dockerfile

```diff
@@ -27,10 +45,12 @@ RUN cat Gemfile.lock | grep -A1 "BUNDLED WITH" | tail -n1 | awk '{print $1}' > B
 RUN set -ex && \
   apk add --update --no-cache make gcc libc-dev mariadb-dev postgresql-dev sqlite-dev git && \
   apk upgrade && \
+  gem uninstall bundler -v 2.1.4 && \
   gem install bundler -v $(cat BUNDLER_VERSION) && \
-  ls /usr/local/lib/ruby/gems/2.7.0 && \
-  gem install rdoc -v "6.3.2" --install-dir /usr/local/lib/ruby/gems/2.7.0 && \
-  gem uninstall --install-dir /usr/local/lib/ruby/gems/2.7.0 -x rake && \
+  ls /usr/local/lib/ruby/gems/${RUBY_VERSION} && \
+  gem install rdoc -v "6.3.2" --install-dir /usr/local/lib/ruby/gems/${RUBY_VERSION} && \
+  gem uninstall --install-dir /usr/local/lib/ruby/gems/${RUBY_VERSION} -x rake && \
+  rm /usr/local/lib/ruby/gems/${RUBY_VERSION}/specifications/default/bundler-2.1.4.gemspec && \
```

Comment:
Trivy complained about bundler 2.1.4 in the image, but didn't in CI

https://github.com/YOU54F/pact-broker-docker/actions/runs/4736414697

explicitly at this path